services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )

```

Comment:
Ensure that the `create_user` function handles potential exceptions during user creation, such as validation errors. Consider wrapping the user creation in a try-except block to manage these cases.